scripts/toggle.sh

```diff
@@ -15,7 +15,7 @@ while getopts :-:BCEb:c:d:e:h:s:S:t:T:w:x:y: OPT; do
 	[BCE]) popup_args+=("-$OPT") ;;
 	[bcdhsStTwxy]) popup_args+=("-$OPT" "$OPTARG") ;;
 	# forward environment overrides to popup sessions
-	e) session_args+=("-e" "$OPTARG") ;;
+	e) session_args+=("-e" "'$OPTARG'") ;;
```

Comment:
I prefer to simply use `$(escape "${session_args[@]}")` like `$(escape "${cmd[@]}")` in <https://github.com/cenk1cenk2/tmux-toggle-popup/blob/7196e6c5fa73e6271c96f4a1fdaaa5fb3b35d50e/scripts/toggle.sh#L82>.